integration-cli/docker_cli_run_test.go

```diff
@@ -4536,3 +4536,17 @@ func (s *DockerDaemonSuite) TestRunWithUlimitAndDaemonDefault(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(out, checker.Contains, "[nofile=42:42]")
 }
+
+func (s *DockerSuite) TestRunStoppedLoggingDriverNoLeak(c *check.C) {
+	nroutines, err := getGoroutineNumber()
+	c.Assert(err, checker.IsNil)
+
+	out, _, err := dockerCmdWithError("run", "--name=fail", "--log-driver=fluentd", "busybox", "true")
+	c.Assert(err, checker.NotNil)
+	if !strings.Contains(out, "Failed to initialize logging driver") {
```

Comment:
Sure, I tried to look for example in this file and failed to find it. Thanks for pointing out.